orc-rt/include/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(files
+    orc-rt-c/orc-rt.h
+)
+
+add_library(orc-rt-headers INTERFACE)
+target_include_directories(orc-rt-headers INTERFACE
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>
+)
+target_sources(orc-rt-headers
+    INTERFACE FILE_SET HEADERS
+    BASE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}
+    FILES ${files}
+)
```

Comment:
File Sets were introduced in CMake 3.23 but LLVM currently requires 3.20 as the minimum version.

Reply:
Thanks for spotting that! I've switched to `install (DIRECTORY ...` in the latest commits -- Is that a reasonable solution for CMake 3.20?